angular-chart.js

```diff
@@ -194,9 +194,10 @@
     }
 
     function getColours (type, scope) {
-      var colours = angular.copy(scope.colours ||
-        Chart.defaults.global.colours ||
-        ChartJs.getOptions(type).colours);
+      var colours = angular.copy(ChartJs.getOptions(type).colours ||
+        scope.colours ||
```

Comment:
I think it should be 

1. scope.colours: this is the more specific
2. ChartJs.getOptions(type).colours): default options set using `setOptions`
3. Chart.defaults.global.colours: default global options

WDYT?